app/interfaces/api/v2/ccr_claim.rb

```diff
@@ -5,7 +5,12 @@ class CCRClaim < Grape::API
 
       helpers do
         def claim
-          ::Claim::AdvocateClaim.find_by(uuid: params[:uuid]) || error!('Claim not found', 404)
+          ::Claim::BaseClaim.agfs.find_by(uuid: params[:uuid]) || error!('Claim not found', 404)
```

Comment:
Related with this, I've been delaying [this PR](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2323) for a while but I think it would beneficial to get it out ASAP.

❓ If the `find_by(uuid: params[:uuid])` gives you a unique record, what's the need for the `agfs` scope?

Reply:
may be its just "belt and braces" but it would prevent an LGFS claim query being sent to the wrong endpoint (ccr endpoint). Same/reverse logic exists for the cclf endpoint - in such instances it would return claim not found.